pkg/build/build.go

```diff
@@ -126,6 +126,8 @@ func (b *Build) Run(ctx context.Context, recreateCluster bool) error {
 	}
 
 	setupLog.Info("Adding CRDs to the cluster")
+	// Wait for API Server to come up
+	time.Sleep(15)
 	if err := b.ReconcileCRDs(ctx, kubeClient); err != nil {
 		return err
 	}
```

Comment:
this could work but better would be if we put a retry mechanism in place. what do you think?

Reply:
Sorry this was for some internal testing things--not a real PR for now. If this is something that's needed we'll do a real mechanism not a sleep.

Reply:
sounds good.